chaoslib/pumba/pumba.j2

```diff
@@ -1,23 +1,22 @@
 
-apiVersion: apps/v1
-kind: DaemonSet
+apiVersion: batch/v1
+kind: Job
 metadata:
   name: pumba
   labels: 
     chaosUID: {{ chaosUID }}
 spec:
-  selector:
-    matchLabels:
-      app: pumba
-      chaosUID: {{ chaosUID }}
   template:
     metadata:
       labels:
         app: pumba
         chaosUID: {{ chaosUID }}
-        com.gaiaadm.pumba: "true" # prevent pumba from killing itself
-      name: pumba
+        # prevent pumba from killing itself
+        com.gaiaadm.pumba: "true"
     spec:
+      nodeSelector:
+        kubernetes.io/hostname: {{ app_node }}
+      restartPolicy: Never
       containers:
       - image: {{ pumba_image }}
```

Comment:
try using 0.6.5 ? (same as network chaos) -- in which case the pumba job yaml should have the args specified in it. kubectl exec will no more be needed